src/plugins/intel_gpu/src/graph/include/resample_inst.h

```diff
@@ -22,7 +22,13 @@ struct typed_program_node<resample> : public typed_program_node_base<resample> {
 
     program_node& input() const { return get_dependency(0); }
 
-    std::vector<size_t> get_shape_infer_dependencies() const override { return {1, 2}; }
+    std::vector<size_t> get_shape_infer_dependencies() const override {
+        // if vector of sizes or scales exists, resample in CreateInterpolateOp generates no dependency for inputs of sizes and scales
+        if (typed_desc()->sizes.size() != 0 || typed_desc()->scales.size() != 0)
```

Comment:
nit: could you add an assertion so that we can prevent mistake in the future?
```
// Both sizes and scales should be fed as attribute
OPENVINO_ASSERT(typed_desc()->sizes.size() != 0 && typed_desc()->scales.size() != 0)
```

Reply:
Applied assertion to prevent mistake

Reply:
Applied assertion to the if condition.